control-plane-operator/controllers/hostedcontrolplane/cvo/reconcile.go

```diff
@@ -303,20 +304,38 @@ func preparePayloadScript(platformType hyperv1.PlatformType, oauthEnabled bool)
 	return strings.Join(stmts, "\n")
 }
 
-func cvoBootrapScript(clusterID string) string {
+func cvoBootstrapScript(clusterID string) string {
+	enabledCaps := sets.New[configv1.ClusterVersionCapability](
+		configv1.ClusterVersionCapabilitySets[configv1.ClusterVersionCapabilitySetCurrent]...)
+	enabledCaps = enabledCaps.Delete(configv1.ClusterVersionCapabilityImageRegistry)
+
+	capabilities := strings.Builder{}
+	capList := enabledCaps.UnsortedList()
+	for i, capability := range capList {
+		capabilities.WriteString(string(capability))
+		if i != len(capList)-1 {
+			capabilities.WriteString(",")
+		}
+	}
+
 	payloadDir := volumeMounts.Path(cvoContainerBootstrap().Name, cvoVolumePayload().Name)
 	var scriptTemplate = `#!/bin/bash
 set -euo pipefail
-cat > /tmp/clusterversion.yaml <<EOF
+cat > /tmp/clusterversion.yaml <<-EOF
 apiVersion: config.openshift.io/v1
 kind: ClusterVersion
 metadata:
   name: version
 spec:
+  capabilities:
+    baselineCapabilitySet: 'None'
+    additionalEnabledCapabilities: [%s]
   clusterID: %s
+
 EOF
 oc get ns openshift-config &> /dev/null || oc create ns openshift-config
 oc get ns openshift-config-managed &> /dev/null || oc create ns openshift-config-managed
+oc create -f /tmp/clusterversion.yaml
```

Comment:
at this point, the resource was not yet deployed:

```
error: resource mapping not found for name: "version" namespace: "" from "/tmp/clusterversion.yaml": no matches for kind "ClusterVersion" in version "config.openshift.io/v1"
ensure CRDs are installed first

```

Reply:
Does the `ClusterVersion` resource need to be created before the stuff in `manifests` to avoid creation of resources for disabled capabilities?